.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+* Version of embedded-graphics in use (if applicable): [version here]
```

Comment:
Knowing the target platform would also be useful in some cases: Simulator or real hardware and which display/driver is used.